python/tk_maya/menu_generation.py

```diff
@@ -245,7 +285,7 @@ def get_documentation_url_str(self):
             app = self.properties["app"]
             doc_url = app.documentation_url
             # deal with nuke's inability to handle unicode. #fail
-            if doc_url.__class__ == unicode:
+            if type(doc_url) == six.text_type:
```

Comment:
The comment mentions Nuke. I think this is a copy paste and can probably be removed.

Reply:
Actually I can't find any usage of this whole method at all, not in this engine, and not in core, or the handful of apps I searched. 
I see most other engines (apart from Photoshop and After Effects) implement a very similar `AppCommand` class, with this method. But again I don't see this method being used by anything. It's not publically documented either, so maybe it should just be removed?

Reply:
If it's not used, ditch it! 😃